### ISSUE TITLE
Problem Compiling JSC using fuzzbuild.sh

### ISSUE THREAD

**Issue by andyzekan** [googleprojectzero/fuzzilli#450]
Hello @saelo ,
I am trying to fuzz JSC. After downloading its source and going through README.md, it does not work. I get an error that the target is terminated with no zero exit code (signal 6). I even tried to apply the patch line by line by hand with no luck! My host is a Debian 12. It is interesting that there is no issue with V8 and spidermonkey. It appears the JSC fuzzbuild.sh is very old. ENABLE_FUZZILLI flag is already introduced in JSC.
I wonder if there is something wrong with my setup or does the script need update? Can you take a look?
Thanks.

**Comment by saelo** [comment]
Yeah I haven't fuzzed JSC for a long time so the support for that is very outdated. Probably a lot of the patch can be deleted since most of it has been upstreamed by now. If you get it to work, it'd be great to update the files in Fuzzilli!

**Comment by andyzekan** [comment]
Thanks for your reply. yeah, it was tricky to set it up, but it was still possible for me to get it work. Unfortunately, recently I have not been able to set it up after several days (I even tried applying patch manually line by line). I am afraid it needs a deeper expert look.
